IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,383 @@
+
+\subsection{Benefit to RIT}
+\label{subsec:Benefit to RIT}
+By attending the event and competing against top universities, RIT will be able to use this as an opportunity to showcase
+how passionate and involved their students are in space exploration. This involvement can be used in media as a tool for
+achieving more funds and attracting new students to the University. Since RIT has not been involved with IREC before, SPEX's attendance at
+this competition can also show how the University and its students are growing and taking on new challenges.
+
+\subsection{Partnership with Launch Initiative}
+\label{subsec:Partnership with Launch Initiative}
+Since the founding of SPEX, there has never been a strong relationship with the Launch Initiative team. By cooperatively participating
+in this event, a great opportunity is presented to create a good relationship between two of the largest space-related teams on
+campus. In the future, united efforts between the two teams could result in more advanced projects which will provide
+even more exposure to both RIT space teams.
+
+\section{Implementation}
+\label{sec:Implementation}
+
+\subsection{Selection}
+\label{subsec:Selection}
+  % What path do you anticipate the project to take?
+A decision matrix was created which accounts for the following factors and assigned a maximum possible
+score based on importance to the selection process:
+\begin{itemize}
+  \item \textbf{Scientific Relevance (10 pts)}
+  \begin{itemize}
+    \item How does this payload contribute to the scientific community?
+  \end{itemize}
+
+  \item \textbf{Potential for Excellent Technical Execution (15 pts)}
+  \begin{itemize}
+    \item How feasible is it for the SPEX team to have great technical success with this endeavor?
+  \end{itemize}
+
+  \item \textbf{Time Required/ Complexity (5 pts)}
+  \begin{itemize}
+    \item Will this payload result in too much time being spent on research and development?
+  \end{itemize}
+
+  \item \textbf{Constructability (5 pts)}
+  \begin{itemize}
+    \item Does this payload require large amounts of machining and assembly or materials that are difficult to work with?
+  \end{itemize}
+
+  \item \textbf{Cost (5 pts)}
+  \begin{itemize}
+    \item How expensive will this be?
+  \end{itemize}
+
+  \item \textbf{Weight (5 pts)}
+  \begin{itemize}
+    \item Will it be easy to meet the 8.8lbs minimum requirement?
+  \end{itemize}
+
+  \item \textbf{Programmability (5 pts)}
+  \begin{itemize}
+    \item Does this payload require the use or creation of advanced software?
+  \end{itemize}
+
+  \item \textbf{"WOW" Factor (5 pts)}
+  \begin{itemize}
+    \item Is this payload attractive? Will it stick out among others?
+  \end{itemize}
+\end{itemize}
+
+Based on the results of the individual scoring of each idea throughout the team, the "Hyperion" payload was selected to move forward with
+an average score of 44 out of a maximum 55 points.
```

Comment:
Why? what were the other ideas? How were points assigned? WHat is the payload even? (I see it below, but right now you havent described it at all)
What steps were taken to avoid/eliminate bias in your decisions?